File: 01. Lenguajes de proposito general/01. Programación básica/08. Funciones/06. Con paramatros y con retorno multiple/Python/Con paramatros y con retorno multiple.py

# Creación con parametros limitados
def funcion(var1, var2, varN) 
    # Contenido
    return valor1, valor2, valorN

# Creación con parametros ilimitados
def funcion(*variables) 
    # Contenido
    return valor1, valor2, valorN

# Creación con parametros por defecto (Los valores por defecto de pueden omitir en el llamado)
def funcion(var1 = def1, var2 = def2, varN = defN) 
    # Contenido
    return valor1, valor2, valorN
 
# Llamado
''' Contexto ''' funcion(valor1, valor2, valorN) # Contexto

# Llamado con asignación
varible1, varible2, varibleN = funcion(valor1, valor2, valorN)

# Eliminación
del funcion

File: 01. Lenguajes de proposito general/01. Programación básica/08. Funciones/06. Con paramatros y con retorno multiple/JavaScript/Con paramatros y con retorno multiple.js

// Creación con parametros limitados
function funcion(var1, var2, varN) {
    // Contenido
    return [valor1, valor2, valorN];
}

// Creación con parametros ilimitados
function funcion(...variables) {
    // Contenido
    return [valor1, valor2, valorN];
}

// Creación con parametros por defecto (Los valores por defecto de pueden omitir en el llamado)
function funcion(var1 = val1, var2 = val2, varN = valN) {
    // Contenido
    return [valor1, valor2, valorN];
}
 
// Llamado
/*Contexto*/ funcion(valor1, valor2, valorN) /*Contexto*/

// Llamado con asignación
[varible1, varible2, varibleN] = funcion(valor1, valor2, valorN);

// Eliminación
NO SOPORTADO (EL GARBAGE COLLECTOR LA ELIMINA AUTOMATICAMENTE CUANDO YA ES INACCESIBLE)
